setup.py

```diff
@@ -44,17 +44,6 @@
     ],
     license='Apache 2.0',
     python_requires='>=3.6',
-    install_requires=[
```

Comment:
What's the rationale behind removing those packages from the setup.py?
`pip install tensortrade` will use this script, and not requirements.txt (which is used more for development\testing purposes)

Reply:
I should have put it back there in a later commit, but pinning the Tensorflow version to avoid bumping the Docker image's version. I had to do that when using Docker unless you have another trick to avoid reinstalling another version of Tensorflow

Reply:
just put it back in